src/wmtk/EdgeMeshOperationExecutor.hpp

```diff
@@ -0,0 +1,66 @@
+#pragma once
+#include <wmtk/utils/Logger.hpp>
+#include "EdgeMesh.hpp"
+#include "SimplicialComplex.hpp"
+#include "Tuple.hpp"
+namespace wmtk {
+class EdgeMesh::EdgeMeshOperationExecutor
+{
+public:
+    EdgeMeshOperationExecutor(EdgeMesh& m, const Tuple& operating_tuple, Accessor<long>& hash_acc);
+    void delete_simplices();
+    void update_cell_hash();
+
+    std::array<Accessor<char>, 2> flag_accessors;
+    Accessor<long> ee_accessor;
+    Accessor<long> ev_accessor;
+    Accessor<long> ve_accessor;
+    Accessor<long>& hash_accessor;
+
+    /**
+     * @brief gather all simplices that are deleted in a split
+     *
+     * The deleted simplices are exactly the open star of the edge
+     */
+    static const std::array<std::vector<long>, 2> get_split_simplices_to_delete(
```

Comment:
Please be more specific in the documentation what is returned (I know the other operation executors are just as badly documented, but please, be better 🙂)